datajs/src/browser-utils/index.js

```diff
@@ -3,5 +3,6 @@ else
   module.exports = {
     toNodeStream: (reader, size, returnChunk) => {},
     isFileFromBrowser: (file) => {},
-    readChunk: (file, next, done) => {},
+    webToNodeStream: (reader) => {},
+    computeHash: (fileStream, fileSize, algorithm) => {}
```

Comment:
I don't think this only browser related function, so don't think we need this in browser-utils. I would even move this to base File class.